lib/generative.rb

```diff
@@ -1,4 +1,7 @@
 require 'rspec/core'
+if RUBY_DESCRIPTION
+  require 'forwardable'
+end
```

Comment:
I think it would be safe to just `require 'forwardable'` without looking at the Ruby description/version